### PR TITLE
Include loss scaling in README code example

### PR DESCRIPTION
**Pull request by f0k**
The code example in the README, although meant to illustrate checkpointing, is the first `amp` usage example seen by a new user, and it looks so complete that it's tempting to just copy what's written there to adapt one's training script. Unfortunately, it misses the loss scaling, so a new user doing so will find that checkpointing works great, but training doesn't work at all. (Been there, done that.)

I'd propose to just add the two lines needed for loss scaling (this PR), or remove the code example completely so people will be forced to look at the documentation.

### CODE REVIEW

**Comment by ptrblck** [comment]
Thanks @f0k for the suggestion and PR!
I would prefer to add the loss scaling code into the example.

CC @mcarilli what do you think?

**Comment by mcarilli** [comment]
Looks good to me, thanks!
